### DISCUSSION TITLE
Vue component is not rendering after adding Octokit

### DISCUSSION THREAD

**Discussion by romaldowoho** [octokit/octokit.js#2238] [Q&A]
Has anyone experienced this? 

I added the following code to connect Octokit in the script of my component and the component just stopped rendering, I don't see it in the elements tab. I see the log in the console with my username, so Octokit connects successfully and it doesn't show any errors.

```import { Octokit } from "octokit";

  const octokit = new Octokit({ auth: `myToken` });

  const {
      data: { login },
  } = await octokit.rest.users.getAuthenticated();
  console.log("Hello, %s", login);
```

**Comment by timrogers** [comment]
👋 If your username is being logged, then code is definitely executing correctly, so it seems unlikely that Octokit specifically is breaking your component rendering. 

I don’t know much about Vue, but I’m wondering if the use of `await` could be part of the issue. 

Would you be able to share the full code for your component so we can see the context? We might not be able to help as this looks like a Vue issue, but I’ll try my best.
